### PR TITLE
DAT-4430 use ui-responsive in search suggestions Mercury vs Oasis

### PR DESCRIPTION
**Pull request by dianafa**
## Links

* http://wikia-inc.atlassian.net/browse/DAT-4430

## Description

Now we can use ui-responsive

## Reviewers

@idradm

### CODE REVIEW

File: front/main/app/components/wikia-search.js [hunk 1]
```diff
@@ -69,13 +72,11 @@ export default Component.extend(NoScrollMixin,
...
 				this.set('inputField', $('.side-search__input'));
-			});
 
-			if (this.get('focusInput')) {
-				run.scheduleOnce('afterRender', this, () => {
-					this.$('.side-search__input').focus();
-				});
-			}
+				if (this.get('focusInput')) {
+					this.get('inputField').focus();
```
Comment (SebastianMarzjan) [inline front/main/app/components/wikia-search.js]: Why was the scheduling removed?

Comment (dianafa) [inline front/main/app/components/wikia-search.js]: we were scheduling two times, now we do it once ant then we check if we should focus input as well

File: front/main/app/templates/components/wikia-search.hbs [hunk 2]
```diff
@@ -29,12 +29,15 @@
 		<ul class="wikia-search__suggestions">
 			{{#each suggestions as |suggestion|}}
 				<li class="wikia-search__item wikia-search__search-suggestion mw-content">
-					{{#link-to 'wiki-page' suggestion.uri action='searchSuggestionClick' class='wikia-search--mobile'}}
-						{{{suggestion.text}}}
-					{{/link-to}}
-					<a class="wikia-search--desktop" href="#" {{action 'redirectToOasis' suggestion.uri}}>
-						{{{suggestion.text}}}
-					</a>
+					{{#if responsive.isMobile}}
```
Comment (SebastianMarzjan) [inline front/main/app/templates/components/wikia-search.hbs]: Do our Discussions breakpoints correspond to ember-responsive's:
`export default {
  mobile:  '(max-width: 768px)',
  tablet:  '(min-width: 769px) and (max-width: 992px)',
  desktop: '(min-width: 993px) and (max-width: 1200px)',
  jumbo:   '(min-width: 1201px)'
};`

Comment (dianafa) [inline front/main/app/templates/components/wikia-search.hbs]: o tutej: https://github.com/Wikia/mercury/pull/2511/files

**Comment by SebastianMarzjan** [comment]
LGTM

**Pull request merged**